conjureup/controllers/newcloud/common.py

```diff
@@ -87,51 +87,57 @@ def save_creds(cloud, credentials):
 def is_lxd_ready():
     """ routine for making sure lxd is configured for localhost deployments
     """
-    if utils.lxd_version() < parse_version('2.9'):
-        return {"ready": False,
-                "msg": "The current version of LXD found on this system is "
-                "not compatible. Please run the following to get the latest "
-                "supported LXD:\n\n"
-                "  sudo apt-add-repository ppa:ubuntu-lxc/lxd-stable\n"
-                "  sudo apt-get update\n"
-                "  sudo apt-get install lxd lxd-client\n\n"
-                "Or if you're using the snap version:\n\n"
-                "  sudo snap refresh lxd\n\n"
-                "Once complete please re-run conjure-up."}
-
-    if not utils.check_user_in_group('lxd'):
-        return {"ready": False,
-                "msg": "User {} is not part of the LXD group. You will need "
-                "to exit conjure-up and do one of the following:\n\n"
-                " 1: Run `newgrp lxd` and re-launch conjure-up\n\n"
-                "  Or\n\n"
-                " 2: Log out completely, Log in and "
-                "re-launch conjure-up".format(os.environ['USER'])}
-
     try:
-        setup_lxdbr0_network()
-        setup_conjureup0_network()
+        lxd_init()
+        app.log.debug("LXD is configured.")
     except Exception as e:
-        return {"ready": False,
-                "msg": "Unable to determine an existing LXD network bridge, "
-                "please make sure you've run `sudo lxd init` to configure "
-                "LXD.\n\n{}".format(e)}
-
-    if utils.lxd_has_ipv6():
-        return {"ready": False,
-                "msg": "The LXD bridge has IPv6 enabled. Currently this is "
-                "unsupported by conjure-up. Please disable IPv6 and "
-                "re-launch conjure-up\n\n"
-                "Visit https://docs.ubuntu.com/conjure-up/en/troubleshoot#lxd "
-                "for information on how to disable IPv6."}
-
-    app.log.debug("Found an IPv4 address, "
-                  "assuming LXD is configured.")
-    return {"ready": True, "msg": ""}
-
-
-def setup_conjureup0_network():
-    """ This attempts to setup LXD network bridge for conjureup if not available
+        raise
```

Comment:
If we're just unconditionally re-raising, might as well drop the `try` / `except` altogether.